src/test/codegen/atomic-operations-llvm-12.rs

```diff
@@ -0,0 +1,85 @@
+// Code generation of atomic operations for LLVM 12
+// ignore-llvm-version: 13 - 99
+// compile-flags: -O
+#![crate_type = "lib"]
+#![allow(invalid_atomic_ordering)]
```

Comment:
Looks like this is no longer necessary, now that https://github.com/rust-lang/rust/pull/99410 is merged.
```suggestion
```